redash/handlers/users.py

```diff
@@ -194,6 +195,16 @@ def post(self, user_id):
         try:
             self.update_model(user, params)
             models.db.session.commit()
+
+            @after_this_request
+            def update_identity_cookie(response):
+                session['user_id'] = user.get_id()
+
+                remember_token = request.cookies.get('remember_token')
+                if remember_token:
+                    response.set_cookie('remember_token', re.sub('.*\\|', user.get_id() + '|', remember_token))
```

Comment:
I think that you can simply call `login_user` here instead of manually updating things. It will take care of updating the `user_id` and the remember me cookie.

Reply:
Good point regarding updating the session. However with this approach the `remember_token` value is not updated. Not sure what the implications of this though.